src/pages/Checkout/CheckoutPage.vue

```diff
@@ -528,6 +526,22 @@ export default {
 			return hasCredits && isMatchedLoan;
 		});
 		this.matchedText = matchedLoansWithCredit[0]?.loan?.matchingText ?? '';
+
+		this.apollo.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } });
```

Comment:
This assignment query should be needed here, since it's called in the preFetch.

Reply:
Is not working if removed

Reply:
What isn't working? I just tried commenting out this line, and a version was successfully returned by the `readFragment` on the next line. The `preFetch` is called before this `create` method, so the assignment should be in the apollo cache.

Reply:
It was not working to me when removing; makes sense what you mentioned, lemme try again...